build/CRD.Dockerfile

```diff
@@ -0,0 +1,14 @@
+#
```

Comment:
build folder should not contain any code. Please create separate folder similar to other components

Reply:
done